mix.exs

```diff
@@ -51,7 +51,7 @@ defmodule Ueberauth.Github.Mixfile do
   defp package do
     [
       description: "An Ueberauth strategy for using Github to authenticate your users.",
-      files: ["lib", "mix.exs", "README.md", "LICENSE"],
+      files: ["lib", "mix.exs", "README.md", "LICENSE.md"],
```

Comment:
Why do we need this change? The file doesn't have that extension.

![image](https://user-images.githubusercontent.com/73386/133663193-a6af4e5e-8fa8-42e5-aba5-f7912bb8db32.png)

Reply:
![image](https://user-images.githubusercontent.com/73386/133663581-cd3bbb82-a051-4c0c-8791-a74f0d91b021.png)

There was another merge related to this. I'd really like to understand this change before we just start pushing changes chasing this.

Reply:
I am trying to fix CI, I made a mistake  and I am trying to fix, my apologies